Makefile

```diff
@@ -202,3 +202,7 @@ JSONNETFMT_CMD := $(JSONNETFMT) -n 2 --max-blank-lines 2 --string-style s --comm
 .PHONY: jsonnet-fmt
 jsonnet-fmt: | $(JSONNETFMT)
 	PATH=$$PATH:$(BIN_DIR):$(FIRST_GOPATH)/bin echo ${JSONNET_SRC} | xargs -n 1 -- $(JSONNETFMT_CMD) -i
+
+.PHONY: rules/rules.go
```

Comment:
This one should not be a `.PHONY` target, it produces a real file that can be tracked for updates
```suggestion
```

Reply:
I think it should still be made an explicit dependency of the binary target. Consider what happens if you `rm rules/rules.go`. If you then do `make tmp/bin/observatorium-api`, the wildcard doesn't catch this file because the file does not exist on disk, so the rules.go file is note generated.

Reply:
if I don't make it .PHONY, when changing `spec.yaml` the `rules.go` is not regenerated/updated:
```bash
jessica@cupsofwonder:~/workspace/api|add-rules-api-spec⚡ ⇒  make rules/rules.go
make: 'rules/rules.go' is up to date.
```

same happens when using `make generate`..

good point with making the dependency explicit, will add

Reply:
aha. Yes, this target should _not_ be marked as `.PHONY`. The missing piece here is that the target needs to depend on the `rules/spec.yaml`, this way whenever the spec is newer than the generated file, make will know that the rules.go file needs to be regenerated.

Reply:
nice, tested locally and worked :tada: thanks!